src/shared/components/case-editor/case-edit-page/case-edit-page.html

```diff
@@ -1,5 +1,10 @@
-<!-- Current Page && Event trigger name -->
-<h1 class="heading-h1"><span *ngIf="currentPage && currentPage.label">{{currentPage.label}}&nbsp;-&nbsp;</span>{{ eventTrigger.name}}</h1>
+<ng-container *ngIf="currentPage">
+  <h1 *ngIf="!currentPage.label" class="govuk-heading-l">{{eventTrigger.name}}</h1>
+  <ng-container *ngIf="currentPage.label">
+    <span class="govuk-caption-l">{{ eventTrigger.name}}</span>
```

Comment:
These look to be in a different order to the original of `currentPage.label` followed by `eventTrigger.name` (and also separated by a hyphen). Is the change in order and lack of a hyphen intentional?

Reply:
Yes - it's basically being split in two. If there is no label then the eventTrigger is the heading, and if there is a label the label becomes the heading and the eventTrigger becomes the gov-uk caption that sits above the heading.